aptos-move/replay-benchmark/src/commands/initialize.rs

```diff
@@ -84,16 +78,18 @@ impl InitializeCommand {
         })?;
 
         // TODO:
-        //  Right now, only features can be overridden. In the future, we may want to support:
-        //      1. Framework code, e.g., to test performance of new natives or compiler,
-        //      2. Gas schedule, to track the costs of charging gas or tracking limits.
-        //      3. BlockExecutorConfigFromOnchain to experiment with different block cutting based
-        //         on gas limits.
+        //   1. Override gas schedule, to track the costs of charging gas or tracking limits.
+        //   2. BlockExecutorConfigFromOnchain to experiment with different block cutting based
+        //      on gas limits?.
+        //   3. Build options for package overrides.
+        let build_options = BuildOptions::move_2();
+        let package_override = PackageOverride::new(self.override_packages, build_options)?;
```

Comment:
It could make sense to pass these to OverrideConfig constructor, and create package_override there inside the constructor?